paddle/platform/nccl_test.cu

```diff
@@ -86,6 +86,8 @@ TEST(NCCL, all_reduce) {
 
   VLOG(1) << "Invoking ncclAllReduce";
 
+  dynload::ncclGroupStart();
```

Comment:
Is this `ncclGroupStart ` and `ncclGroupEnd` should be added before any NCCL calls? Or only once after `ncclCommInitAll `?

The docs didn't tell when it should be added.